src/amp-story-player/amp-story-entry-point/amp-story-entry-point-impl.js

```diff
@@ -56,8 +77,11 @@ export class AmpStoryEntryPoint {
       return;
     }
 
-    this.initializeShadowRoot_();
+    const playerId = this.element_.getAttribute('player-id');
+    this.player_ = this.doc_.querySelector(`#${playerId}`);
+    this.stories_ = toArray(this.player_.querySelectorAll('a'));
```

Comment:
The player element now has a `getStories` method returning story objects, so I think we shouldn't use the anchors as the source for this.

Reply:
Not available in this PR. We might also not want to do that because we don't want the entry point to wait till after the player is loaded.